pkgs/development/python-modules/screed/default.nix

```diff
@@ -32,11 +32,12 @@ buildPythonPackage rec {
     "Test_fq_shell_command"
   ];
 
-  propagatedBuildInputs = [ bz2file setuptools ];
+  propagatedBuildInputs = [ setuptools ];
```

Comment:
Maybe `setuptools` can auto be removed? Upstream has replaced uses of `pkg_resources`.

Reply:
you're absolutely right, the whole point of making the upstream release was removing `setuptools` as a runtime dep!